### PR TITLE
Formatting and line lengths on source

### PR DESCRIPTION
**Pull request by mprorock**
***
<a href="https://pr-preview.s3.amazonaws.com/mesur-io/did-imp-guide/pull/24.html" title="Last updated on Aug 24, 2021, 10:51 AM UTC (9471aa9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-imp-guide/24/0e16f40...mesur-io:9471aa9.html" title="Last updated on Aug 24, 2021, 10:51 AM UTC (9471aa9)">Diff</a>

### CODE REVIEW

**Comment by mprorock** [comment]
still noticing a few items on the link checker complaining about fragments that do resolve - so still looking into that - appears to only be an issue on full link checking

**Review by OR13 — APPROVED** [review]

**Comment by OR13** [comment]
ping @peacekeeper this PR is big, could us an extra review.

**Comment by mprorock** [comment]
> ping @peacekeeper this PR is big, could us an extra review.

yes, should be very few actual content changes (primarily restricted to capitalization and link formatting) - but brings everything to 80 chars and in standard html lint formatting so we have a good baseline rolling forward

File: index.html [hunk 1]
```diff
@@ -179,13 +181,15 @@ <h2>Methods</h2>
     </p>
 
     <p>
-      Avoid constructing a new DID Method that is nearly identical to an existing
+      Avoid constructing a new DID Method that is nearly identical to an
+      existing
       DID Method.
```
Comment (TallTed) [inline index.html:186]:
```suggestion
      existing DID Method.
```

File: index.html [hunk 2]
```diff
@@ -316,7 +323,8 @@ <h2>Verifiable Data Registry</h2>
       <h2>Establishing a DID</h2>
 
       <p>
-        In order to create secure DID systems, it is important to consider the process
+        In order to create secure DID systems, it is important to consider the
+        process
         of establishing a Decentralized Identifier.
```
Comment (TallTed) [inline index.html:328]:
```suggestion
        process of establishing a Decentralized Identifier.
```

File: index.html [hunk 3]
```diff
@@ -327,323 +335,350 @@ <h2>Establishing a DID</h2>
...
-        <li>Use the first verification method to perform a <a data-cite="DID-CORE#method-operations">DID Create operation</a>.</li>
+        <li>Use the first verification method to perform a <a
+            data-cite="DID-CORE#method-operations">DID Create
+            operation</a>.</li>
       </ol>
       <p class="note">
-        The Create Operation will often use the first verification method directly, or a deterministic
+        The Create Operation will often use the first verification method
+        directly, or a deterministic
         function thereof, such as a hash, to construct and register the DID.
```
Comment (TallTed) [inline index.html:353]:
```suggestion
        directly, or a deterministic function thereof, such as a hash, to construct
        and register the DID.
```

File: index.html [hunk 4]
```diff
@@ -327,323 +335,350 @@ <h2>Establishing a DID</h2>
...
       <p class="note">
         True randomness is <b>hard</b> to get. Public sources such as the
-        <a href="https://csrc.nist.gov/projects/interoperable-randomness-beacons">NIST Randomness Beacon</a>
-        are reliable but not appropriate for use directly as a secret key.  See the
+        <a
+          href="https://csrc.nist.gov/projects/interoperable-randomness-beacons">NIST
+          Randomness Beacon</a>
+        are reliable but not appropriate for use directly as a secret key. See
+        the
         section on <a href="#randomness">Randomness</a> for more information.
```
Comment (TallTed) [inline index.html:362]:
```suggestion
        the section on <a href="#randomness">Randomness</a> for more information.
```

File: index.html [hunk 5]
```diff
@@ -36,8 +38,8 @@
 
       // if there is a previously published draft, uncomment this and set its YYYY-MM-DD date
       // and its maturity status
-      previousPublishDate:  "2021-08-26",
-      previousMaturity:  "FPWD",
+      previousPublishDate: "2021-08-26",
+      previousMaturity: "FPWD",
```
Comment (iherman) [inline index.html:42]: The `previousPublishdDate` and `previousMaturity` must be commented out. At this moment, for the FPWD, there is no such previous publication and the result gets a header entry that pubrules rejects (for good reasons...)

File: index.html [hunk 6]
```diff
@@ -661,145 +696,207 @@ <h2>Data Model and Representations</h2>
...
 
-      <pre class="example nohighlight" title="A DID document in the `application/did+json` representation with the `@context` representation-foreign entry">
+        <p>
+          For example, consider the following DID document, and assume that the
+          JSON-LD context
+          `https://myextensions.com/other-definitions.jsonld` defines
+          `blockchainAccountId` in a way that is different
+          from the <a
+            href="https://w3c.github.io/did-spec-registries/#blockchainaccountid">property
+            listed in the DID
```
Comment (iherman) [inline index.html:894]: This should be changed to `https://www.w3.org/TR/did-spec-registries/#blockchainaccountid` (or its `data-cite` equivalent). It is not appropriate to refer to the editor's draft if there is an official /TR document.

File: index.html [hunk 7]
```diff
@@ -815,95 +912,128 @@ <h3>
...
-              href="https://w3c.github.io/DID-SPEC-REGISTRIES/#blockchainaccountid">property listed in the DID
-        Specification Registries</a>, and it will process it accordingly.
-      </p>
+        <p>
+          If this DID document is consumed as `application/did+json` by an
+          implementation that doesn't understand
+          JSON-LD,
+          it will interpret `blockchainAccountId` as the <a
+            href="https://w3c.github.io/did-spec-registries/#blockchainaccountid">property
+            listed in the DID
```
Comment (iherman) [inline index.html:921]: This should be changed to `https://www.w3.org/TR/did-spec-registries/#blockchainaccountid` (or its `data-cite` equivalent). It is not appropriate to refer to the editor's draft if there is an official /TR document.

File: index.html [hunk 8]
```diff
@@ -815,95 +912,128 @@ <h3>
...
-        information about properties as well as representation-specific entries that help with such conversion. For
-        example, a DID document in the `application/did+json` representation can be converted to the
-        `application/did+ld+json` representation by adding a `@context` representation-specific entry during production,
-        using the JSON-LD context information in the DID <a href="https://w3c.github.io/DID-SPEC-REGISTRIES/">Specification
-        Registries</a>.
-      </p>
+        <p>
+          The <a href="https://w3c.github.io/did-spec-registries/">DID
+            Specification Registries</a> provide additional
+          information about properties as well as representation-specific
```
Comment (iherman) [inline index.html:972]: This should be changed to `https://www.w3.org/TR/did-spec-registries` (through a proper external reference). It is not appropriate to refer to the editor's draft if there is an official /TR document.

**Review by iherman — COMMENTED** [review]
W3C's linkechecker (on the version of the document on /TR/2021) has issues with the following reference (beyond what I commented in the text):

`https://csrc.nist.gov/projects/interoperable-randomness-beacons`

which goes to 404. Unless I missed it below I did not see that changed.

(My apologies if this has been changed and I missed it.)

**Comment by mprorock** [comment]
merged updates from @iherman to my branch and merging so that we can get this up and out.

**Pull request merged**